### PR TITLE
add cv

### PR DESCRIPTION
**Pull request by RominaFior**

### CODE REVIEW

File: staff/RominaFior/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <title>CV</title>
+</head>
+<body>
+    <H1>Curriculum Vitae</H1>
+    <h2>Jesica Romina Sanchez</h2>
+    <p>Co Founder at Eqosis</p>
+    <p>Madrid y alrededores, EspañaServicios financieros</p>
+    
+    <h3>Extracto</h3>
+    <p>Internationally focused, strategic and results oriented professional, possessing more than a decade of professional experience in Australia, UK, and in Europe. Primarily experienced in financial, operational and risk analysis spanning across a broad range of geographic and demographic client profiles in both developed and emerging markets. Proven track record in project management through utilising analytical, leadership, communication and time management skills in order to develop and implement business strategy and position organisations for long term success.</p>
```
Comment (alexgbor) [inline staff/RominaFior/cv/index.html]: Everything's in Spanish besides this summary. FIX IT 🗡

Comment (RominaFior) [reply staff/RominaFior/cv/index.html]: Done 👍

Comment (doboapps) [reply staff/RominaFior/cv/index.html]: 👍 👌

**Comment by RominaFior** [comment]
Done !

**Pull request merged**